src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -115,14 +116,19 @@ public void testToString() {
     }
 
     @Test
-    public void testKeyEncryption() {
+    public void testKeyEncryption() throws InvalidParametersException {
         String msg = "Test decryption";
         String decrptedMsg;
 
         decrptedMsg = StringHelper.decrypt(StringHelper.encrypt(msg));
         assertEquals(msg, decrptedMsg);
     }
 
+    @Test(expectedExceptions = { InvalidParametersException .class })
```

Comment:
There is a space between `InvalidParametersException` and `.class`.
Is that intended?

Also, should there be a space after `{` and before `}`?